protocol/signalfx/trace_zipkin.go

```diff
@@ -78,69 +77,72 @@ func (is *InputSpan) fromZipkinV2() (*trace.Span, error) {
 	}
 	is.Span.ParentID = normalizeParentSpanID(is.Span.ParentID)
 
-	return &is.Span, nil
+	return &is.Span
 }
 
-func (is *InputSpan) addParentChildSpanReferenceToJaegerSpan(span *jaegerpb.Span) error {
+func (is *InputSpan) addParentChildSpanReferenceToJaegerSpan(span *jaegerpb.Span) {
 	// only add parent/child reference if the parent id can be parsed
-	parentID, err := jaegerpb.SpanIDFromString(*is.Span.ParentID)
-	if err == nil {
+	if parentID, err := jaegerpb.SpanIDFromString(*is.Span.ParentID); err == nil {
```

Comment:
what to do about the error if parentId cannot be parsed

Reply:
the field isn't required or a violation, if it's garbage, we get rid of it.